i18n/src/main/resources/authorized_role.properties

```diff
@@ -96,8 +96,18 @@ authorizedRole.moderator.table.delete=Delete
 
 authorizedRole.moderator.table.message.popup.headline=Reporters message
 
+authorizedRole.moderator.banDialog.headline=Ban User
```

Comment:
The headline should be more the prompt to what to do. e.g. `Enter a ban reason`
Can be removed then from `authorizedRole.moderator.banDialog.instructions` and maybe renamed to `authorizedRole.moderator.banDialog.details`